charts/rekor/values.yaml

```diff
@@ -93,6 +93,12 @@ server:
     httpGet:
       port: 3000
       path: /ping
+  sharding:
+    name: sharding-config
```

Comment:
do we want to add a bit to allow enabling/disabling sharding?

Reply:
The sharding configmap is required to exist, even if it's empty. There isn't really an option to "disable" it.

Reply:
ack, thanks for the clarification.